package.json

```diff
@@ -25,6 +25,7 @@
     "packages/adapter",
     "packages/language/core/v1",
     "packages/language/examples",
+    "packages/language/spec-cli",
```

Comment:
please, let's call it `specification-client` instead of `spec-cli`

Reply:
done